RetailCoder.VBE/UI/Settings/GeneralSettingsViewModel.cs

```diff
@@ -68,6 +68,14 @@ public ObservableCollection<HotkeySetting> Hotkeys
             }
         }
 
+        public bool ShouldDisplayHotkeyModificationLabel
```

Comment:
Why `Should`? `DisplayHotkeyModificationLabel` or `HotkeyModificationLabelVisible` are more explicit